views/partials/content-layouts/slideshow_gallery.twig

```diff
@@ -11,7 +11,7 @@
                     {% for slide in layout.gallery %}
                         <div class="swiper-slide">
                             {% set slideImage = get_image(slide.id) %}
```

Comment:
You should be able to just pass `slide` to the image partial as it does the get_image function there

Reply:
updated